olwebsite-app/src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Header() {
+  return (
+    <header>
+      <h1>one leif</h1>
```

Comment:
`oneleif`

Reply:
Alright, I'll update it. Did you end up seeing it? If not I can try and post a screen shot

Reply:
Ya! I saw it! I'll post a screenshot. 👍